facedetector/src/main/java/husaynhakeem/io/facedetector/FaceBoundsOverlay.kt

```diff
@@ -46,6 +47,7 @@ class FaceBoundsOverlay @JvmOverloads constructor(ctx: Context, attrs: Attribute
             canvas.drawId(faceBounds.id.toString(), faceBounds.box.center())
             canvas.drawBounds(faceBounds.box)
         }
+        Log.e("faces", "draw ${facesBounds.size}")
```

Comment:
Adding a log message here makes sense, but instead of an error log message, it should be an info or debug message, so `Log.d` or `Log.i`. Also, could you add a static log tag to the class, something like:

``` kotlin
companion object {
    private const val TAG = "FaceBoundsOverlay"
}
```

Then use it for the logging.

``` kotlin
Log.d(TAG, "Drawing ${facesBounds.size}")
```